spec/support/ingestion.rb

```diff
@@ -0,0 +1,23 @@
+module Ingestion
```

Comment:
This is a testing utility for running the ETL process locally on a given record to setup data for automated tests.